app/services/get_stormglass_api_service.rb

```diff
@@ -59,4 +70,47 @@ def call
       )
     end
   end
+
+  private
+
+  def get_rating(wave_height, wave_period, wind_speed)
+    if wave_height <= 0.5
+        wave_rating = 0
+    elsif wave_height <= 1
+        wave_rating = 1
+    elsif wave_height <= 2
+        wave_rating = 2
+    elsif wave_height <= 3.5
+        wave_rating = 3
+    elsif wave_height > 3.5
+        wave_rating = 1
+    end
+
+    
+    if wave_period <= 5
+        period_rating = -3
+    elsif wave_period <= 7
+        period_rating = -2
+    elsif wave_period <= 8
```

Comment:
i generally use around 10+second periods as a cut-off for half decent surf. With that in mind I would be tempted to use something like 8s and 12s as your marker cases.